libr/core/meson.build

```diff
@@ -1,4 +1,5 @@
 r_core_sources = [
+  'rvc.c',
```

Comment:
uhm... i think rvc should be in libr/util, not depend on core at all

Reply:
Tried to do so, it fails apparently r_hash is not usable in util. Is this a problem or should I include it?

Reply:
nope, util cant use hash because of dependencies.. maybe we should move rvc out of util.